bentoml/_internal/bento/bento.py

```diff
@@ -137,6 +137,7 @@ def create(
         build_config: BentoBuildConfig,
         version: t.Optional[str] = None,
         build_ctx: t.Optional[str] = None,
+        tag: t.Optional[t.Union[str, Tag]] = None,
```

Comment:
The name in the tag should always be the bentoml.Service name specified in code, right? I think it make sense to pass version here, but tag is a bit confusing to me

Reply:
On second thought I think you're right; I misunderstood what the docker tag does.

@bojiang did you have something to add here?

Reply:
Maybe we should name the `bentoml containerize .. —tag` option to `—docker-image-tag`?

Reply:
Maybe we should allow for CLI labeling though?

Reply:
> Maybe we should name the `bentoml containerize .. —tag` option to `—docker-image-tag`? Or add an alias?

I think it makes sense as is. Do we expose the name tag in the documentation?

Reply:
> The name in the tag should always be the bentoml.Service name specified in code, right? I think it make sense to pass version here, but tag is a bit confusing to me
Shall make it customizable?

Reply:
I think it will be confusing to users. Better to just name it in one place only. Do you see a use case for it?